grove/qft/tests/test_inverse_qft.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+#
+#    A collection of tests for the inverse quantum Fourier transform.
+#
+#    Written by Aaron Vontell on January 28th, 2017
```

Comment:
Our policy is to just copy the standard header that is all of the other files.  This means that we don't track authorship in source, but you will get to live in infamy here! https://github.com/rigetticomputing/grove/graphs/contributors